src/vlogger/vlogger.cpp

```diff
@@ -48,14 +49,16 @@
 #define VLOG_DEFAULT_MODULE_NAME "XLIO"
 #define XLIO_LOG_CB_ENV_VAR      "XLIO_LOG_CB_FUNC_PTR"
 
+using namespace std::chrono;
```

Comment:
probably not needed since its in the header, right?

Reply:
We can remove this line but then the following line:
`time_point<steady_clock> g_vlogger_usec_on_startup;`
will look as follows:
`std::chrono::time_point<std::chrono::steady_clock> g_vlogger_usec_on_startup;`
Is this better?

Reply:
no, better to use namespace. just wondered if we need it here at all... nevermind